depthai_helpers/calibration_utils.py

```diff
@@ -144,615 +176,749 @@ def calibrate(self, filepath, square_size, out_filepath, flags):
                 print()
                 print(h2)
 
-        # self.create_save_mesh()
+        print("\tTook %i seconds to run image processing." %
+              (round(time.time() - start_time, 2)))
+
+        self.create_save_mesh()
+
+        if self.calibrate_rgb:
+            return self.test_epipolar_charuco_lr(filepath), self.test_epipolar_charuco_rgbr(filepath), self.calib_data
+        else:
+            return self.test_epipolar_charuco_lr(filepath), None, self.calib_data
         
-        # append specific flags to file
-        with open(out_filepath, "ab") as fp:
-            fp.write(bytearray(flags))
-
-        print("Calibration file written to %s." % (out_filepath))
-        print("\tTook %i seconds to run image processing." % (round(time.time() - start_time, 2)))
-        # show debug output for visual inspection
-        print("\nRectifying dataset for visual inspection using Mesh")
-        self.show_rectified_images_two_calib(filepath, False)
-        print("\nRectifying dataset for visual inspection using Two Homography")
-        self.show_rectified_images_two_calib(filepath, True)
-
-    def process_images(self, filepath):
-        """Read images, detect corners, refine corners, and save data."""
-        # Arrays to store object points and image points from all the images.
+
+    def parse_frame(self, frame, stream_name, file_name):
+        file_name += '.png'
+        # filename = image_filename(stream_name, self.current_polygon, self.images_captured)
+        # print(self.data_path + "/{}/{}".format(stream_name, file_name))
+        ds_path = self.data_path + "/{}".format(stream_name)
+        print(ds_path)
+        if not os.path.exists(ds_path):
+            os.makedirs(ds_path)
```

Comment:
can be simplified

```
ds_path = (Path(self.data_path) / stream_name).with_suffix('.png')
ds_path.mkdir(parents=True, exists_ok=True)
```

Reply:
I removed this. There is no use for this.